rust/src/shards/gui/misc/mod.rs

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright © 2022 Fragcolor Pte. Ltd. */
+
+use crate::core::registerShard;
+use crate::types::ExposedTypes;
+use crate::types::ParamVar;
+
+struct GetClipboard {
+  parents: ParamVar,
+  requiring: ExposedTypes,
+}
+
+struct SetClipboard {
+  parents: ParamVar,
+  requiring: ExposedTypes,
+}
+
+struct Reset {
+  parents: ParamVar,
+  requiring: ExposedTypes,
+}
+
+mod clipboard;
+mod reset;
+
+pub fn registerShards() {
+  // registerShard::<GetClipboard>();

```

Comment:
Meant to be disabled?

Reply:
Yes because it doesn't work. Unfortunately, in `egui` the `context.copied_text` is cleared each frame so we can't use that to "read" the clipboard. We would have to request a native call to get the system clipboard. Since we might never need that shard, I won't spend time on it until we do.